test/athens/parser_test.clj

```diff
@@ -28,6 +34,18 @@
     "it’s **very** important"))
 
 
+(deftest parser-pre-formatted-tests
+  (are [x y] (= x (parse-to-ast y))
+    [:block "Hello " [:pre-formatted "world"]]
+    "Hello `world`"
+
+    [:block "Hello " [:pre-formatted "Mars"]]
+    "Hello ```Mars```"
+
+    [:block "Hello " [:pre-formatted "world"] " and " [:pre-formatted "Mars"]]
+    "Hello `world` and `Mars`"))
```

Comment:
Please add one of these and any new markup features to `athens.devcards.parser.cljs`.